### DISCUSSION TITLE
ReactiveUI.Routing functionality / best practices

### DISCUSSION THREAD

**Discussion by jwilburn** [reactiveui/ReactiveUI#3183] [Q&A] [answered]
I'm looking to leverage `ReactiveUI.Routing` to create a wpf wizard / setup type application. Based off the documentation, it looks like you need one view model per view. Does `ReactiveUI.Routing` support one view model per multiple views? The view model code for the _wizard_ could logically be one class instead of being split up.

**Comment by glennawatson** [comment, marked as answer]
Not with the RxUI routing out of the box. It's a bit primitive. In the ideal world, you could pass in a string to indicate the view you want to show.

We also have sextant in development to eventually replace all of RxUI navigation https://github.com/reactiveui/sextant to solve this particular type of issue.

The WPF support is not in yet and still WIP but may be worth following that project

**Reply by jwilburn** [reply]
@glennawatson appreciate the response! I don't think its a deal breaker, I just wanted to make sure I wasn't missing something. I'll keep an eye on sextant
